end-to-end-tests/create_localdb.sh

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bash
```

Comment:
please make sure you get a review from someone else about all the non-js build related stuff, because I have no idea

Reply:
Sigh, I have no idea how this piece of code ended up here. I still mess up quite regularly with git. I have removed this code since not relevant for this PR.